src/v/redpanda/application.cc

```diff
@@ -217,9 +217,13 @@ void application::configure_admin_server() {
           conf.admin_api_doc_dir(), "/v1");
         _admin
           .invoke_on_all([this, rb](ss::http_server& server) {
+              auto insert_comma = [](ss::output_stream<char>& os) {
+                  return os.write(",\n");
+              };
               rb->set_api_doc(server._routes);
               rb->register_api_file(server._routes, "header");
               rb->register_api_file(server._routes, "config");
+              rb->register_function(server._routes, insert_comma);
```

Comment:
why is the comma needed between config and raft, and not between each `register_api_file`?

Reply:
oh i see probably because header isn't really defining any api.

Reply:
header sets up the start of a list, so it's the odd one out:
```json
{
  "paths": {
```
The footer is automagic:
```json
  },
  "definitions": {}
}
```